pom.xml

```diff
@@ -0,0 +1,112 @@
+        <java.version>1.8</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.compiler.version>3.6.1</maven.compiler.version>
+        <groovy.version>2.4.9</groovy.version>
+        <gmavenplus.version>1.6.1</gmavenplus.version>
+        <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
+    </properties>
+
+    <scm>
+        <connection>scm:git:ssh://git@github.com:redhat-cop/pipeline-library.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com:redhat-cop/pipeline-library.git</developerConnection>
+        <url>https://github.com/redhat-cop/pipeline-library.git</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>2.5.0</version>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>1.6.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>addSources</goal>
+                            <goal>addTestSources</goal>
+                            <goal>compile</goal>
+                            <goal>compileTests</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <invokeDynamic>true</invokeDynamic>
+                    <sources>
+                        <source>
+                            <directory>${project.basedir}/src</directory>
+                            <includes>
+                                <include>**/*.groovy</include>
+                            </includes>
+                        </source>
+                        <source>
+                            <directory>${project.basedir}/vars</directory>
+                            <includes>
+                                <include>**/*.groovy</include>
+                            </includes>
+                        </source>
+                    </sources>
+                    <testSources>
+                        <testSource>
+                            <directory>${project.basedir}/test/groovy</directory>
+                            <includes>
+                                <include>**/*.groovy</include>
+                            </includes>
+                        </testSource>
+                    </testSources>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
`<version>...</version>`?
```
$ mvn clean install                                                                                                                                                                                                      
[INFO] Scanning for projects...                                                                                                                                                                                                                
[WARNING]                                                                                                                                                                                                                                      
[WARNING] Some problems were encountered while building the effective model for org.redhatcop:pipeline-library:jar:1.3.0-SNAPSHOT                                                                                                              
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-compiler-plugin is missing. @ line 80, column 21                                                                                                                   
[WARNING]                                                                                                                                                                                                                                      
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.                                                                                                                                    
[WARNING]                                                                                                                                                                                                                                      
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```